public/app/features/alerting/unified/components/amnotifications/AmNotificationsAlertsTableRow.tsx

```diff
@@ -0,0 +1,140 @@
+import { AlertmanagerAlert, AlertState } from 'app/plugins/datasource/alertmanager/types';
+
+import React, { useState } from 'react';
+import { GrafanaTheme2, intervalToAbbreviatedDurationString, Labels } from '@grafana/data';
+import { AmAlertStateTag } from '../silences/AmAlertStateTag';
+import { AlertLabels } from '../AlertLabels';
+import { CollapseToggle } from '../CollapseToggle';
+import { LinkButton, useStyles2 } from '@grafana/ui';
+import { css } from '@emotion/css';
+import { makeAMLink } from '../../utils/misc';
+
+interface Props {
+  className?: string;
+  alert: AlertmanagerAlert;
+  alertManagerSourceName: string;
+}
+
+export const AmNotificationsAlertsTableRow = ({ alert, alertManagerSourceName, className }: Props) => {
+  const [isCollapsed, setIsCollapsed] = useState<boolean>(true);
+  const styles = useStyles2(getStyles);
+
+  const getMatcherQueryParams = (labels: Labels) => {
+    return `matchers=${encodeURIComponent(
+      Object.entries(labels)
+        .map(([labelKey, labelValue]) => {
```

Comment:
```suggestion
        .filter(([labelKey]) => !(labelKey.startsWith('__') && labelKey.endsWith('__'))).map(([labelKey, labelValue]) => {
```
should filter out "internal" labels that start and end with `__`